grails-app/controllers/org/pih/warehouse/inventory/StockTransferController.groovy

```diff
@@ -99,18 +100,26 @@ class StockTransferController {
 
         try {
             stockTransferService.deleteStockTransfer(params.orderId ?: params.id)
-        } catch (IllegalArgumentException e) {
-            flash.message = "${warehouse.message(code: 'default.not.found.message', args: [warehouse.message(code: 'inventory.stockTransfer.label', default: 'Stock Transfer'), params.id])}"
+            params.notificationMessage = g.message(
+                    code: 'react.stockMovement.deleted.success.message.label',
+                    body: 'Stock Movement has been deleted successfully',
+            )
+            params.notificationType = NotificationType.SUCCESS
```

Comment:
Going further what I thought about is creating some util class that would have properties `message` and `type`, so it would be easier to control if we ever consider adding additional properties or anything, but it's just a thought which I wanted to bring out.